Automattic/wp-calypso
# Social: Added new plan constants for the v1 plan

## Proposed Changes

* Added the new social v1 plan constants

## Why are these changes being made?


* To be able to checkout the new plans for social 

## Testing Instructions



* Be able to check out the monthly, yearly, and bi-yearly plans. Go to /checkout/jetpack/jetpack_social_v1_monthly on the local calypso or the live build to checkout the plan on this branch to checkout the yearly plan. Finishing checking out successfully.
* Be able  to switch between the yearly/bi-yearly/monthly variants on the checkout page.
* Ensure Jetpack Cloud pricing loads fine by booting up a live Jetpack Cloud branch. Ensure logged-out view works fine as well.

<img width="970" alt="Screenshot 2024-05-29 at 5 11 36 PM" src="https://github.com/Automattic/wp-calypso/assets/6594561/4e1e4a49-66c2-439a-bcf6-511994582970">


## Pre-merge Checklist
